src/tiled/automappingmanager.h

```diff
@@ -134,6 +136,10 @@ private slots:
      * behavior.
      */
     QString mWarning;
+
+    QFileSystemWatcher *mWatcher;
+
+    QString getRulesFileName();
```

Comment:
Naming style: please call this function simply `rulesFileName`. Please also mark it `const`.